src/software/run_simulated_ai_vs_ai.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# The name of this script
+THIS_SCRIPT_FILENAME=$(basename "$0")
+
+# The number of arguments passed to this script
+NUM_ARGS=$#
+
+if [ $NUM_ARGS -ne 1 ]; then
+  echo "Error: Incorrect number of arguments to script"
+  echo "Usage: $THIS_SCRIPT_FILENAME [interface_name]"
+  echo "Choose from the following interfaces:"
+  echo "`ifconfig`"
```

Comment:
you can run `ls /sys/class/net` net here for a more pruned output

Reply:
fixed